can/tests/test_packer_parser.py

```diff
@@ -182,6 +182,43 @@ def test_updated(self):
       if len(user_brake_vals):
         self.assertEqual(vl_all[-1], parser.vl["VSA_STATUS"]["USER_BRAKE"])
 
+  def test_honda_checksum(self):
```

Comment:
can you pull this into a separate file called test_checksums?

Reply:
+1, been meaning to do this for VW. Will do so once a place/structure is settled on.

Reply:
Done. Comments?